Package.swift

```diff
@@ -28,6 +28,8 @@ let package = Package(
                 "CAtomic",
                 .product(name: "cmark-gfm", package: cmarkPackageName),
                 .product(name: "cmark-gfm-extensions", package: cmarkPackageName),
+            ], exclude: [
```

Comment:
Nit: add a new line to align the style

```suggestion
            ], 
            exclude: [
```